Content.Server/_DV/Antag/NukieAutoWarComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Shared._DV.Antag;
+using Robust.Shared.Analyzers;
+
+namespace Content.Server._DV.Antag;
+
```

Comment:
Both using directives unused
```suggestion
namespace Content.Server._DV.Antag;

```